aidial_sdk/chat_completion/base.py

```diff
@@ -10,3 +10,6 @@ async def chat_completion(
         self, request: Request, response: Response
     ) -> None:
         """Implement chat completion logic"""
+
+    async def rate_response(self, request: RateRequest) -> None:
+        pass
```

Comment:
please replace pass to comment like in the completion method for uniformity:

```python
"""Implement chat completion logic"""
```

Reply:
fixed